### PR TITLE
feat: remove address because streetname was removed

### PR DESCRIPTION
**Pull request by pgallik**

### CODE REVIEW

File: src/AddressRegistry.Consumer.Read.StreetName/Projections/StreetNameCommandHandlingProjections.cs [hunk 1]
```diff
@@ -22,11 +22,20 @@ public StreetNameCommandHandlingProjections()
...
+                        FromProvenance(message.Provenance));
+
+                await commandHandler.Handle(command, ct);
+            });
         }
 
         private static Provenance FromProvenance(Contracts.Provenance provenance) =>
             new Provenance(
-                InstantPattern.General.Parse(provenance.Timestamp).GetValueOrThrow(),
+                SystemClock.Instance.GetCurrentInstant(),
```
Comment (jvandaal) [inline src/AddressRegistry.Consumer.Read.StreetName/Projections/StreetNameCommandHandlingProjections.cs:38]: Good catch!

File: src/AddressRegistry/StreetName/Commands/RemoveAddressBecauseStreetNameWasRemoved.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
+namespace AddressRegistry.StreetName.Commands
+{
+    using System;
+    using System.Collections.Generic;
+    using Be.Vlaanderen.Basisregisters.Generators.Guid;
+    using Be.Vlaanderen.Basisregisters.GrAr.Provenance;
+    using Be.Vlaanderen.Basisregisters.Utilities;
+
+    public class RemoveAddressBecauseStreetNameWasRemoved : IHasCommandProvenance
```
Comment (jvandaal) [inline src/AddressRegistry/StreetName/Commands/RemoveAddressBecauseStreetNameWasRemoved.cs:9]: Why this new command? I don't think it is ever used?

Comment (pgallik) [reply src/AddressRegistry/StreetName/Commands/RemoveAddressBecauseStreetNameWasRemoved.cs:9]: Indeed, we changed the implementation and forgot to remove this.

**Review by jvandaal — APPROVED** [review]

**Review by jvandaal — APPROVED** [review]

**Pull request merged**